packages/analytics-node/src/plugins/context.ts

```diff
@@ -0,0 +1,35 @@
+import { BeforePlugin, NodeConfig, Event, PluginType } from '@amplitude/analytics-types';
+import { UUID } from '@amplitude/analytics-core';
+import { VERSION } from '../version';
+
+export class Context implements BeforePlugin {
+  name = 'context';
+  type = PluginType.BEFORE as const;
+
+  // this.config is defined in setup() which will always be called first
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  config: NodeConfig;
+  eventId = 0;
+  library = `amplitude-ts/${VERSION}`;
```

Comment:
The library name should be different than browser one

Reply:
I would suggest `amplitude-node-ts`

Reply:
Good idea. Will update it.